### DISCUSSION TITLE
Name

### DISCUSSION THREAD

**Discussion by jagoosw** [OceanBioME/OceanBioME.jl#3] [Ideas]
Having looked at the [Julia Registry naming guidelines](https://pkgdocs.julialang.org/v1/creating-packages/#Package-naming-guidelines-1) it seems like `Biogeochemistry.jl` or something similar might be the best name choice? Perhaps `OceanBiogeochemistry.jl`to make it a little clearer what it does. 

I don't think its very common in Julia like it is in Python but we could establish the convention that its imported with a pseudoname like `using OceanBiogeochemistry as BGC`if we're concerned with the name being too long as well.

**Comment by johnryantaylor** [comment]
One idea that I had recently is:
OceanBioME.j
which would stand for
Ocean Biogeochemical Modelling Environment. I also kind of like biome because it encapsulates what we’re trying to do (create a self-contained virtual biome). I’m very open to other ideas too.

**Reply by jagoosw** [reply]
I like that name. I would need to check but I think that the Julia registry won't merge names with a capital letter as the last character, but you can get them todo it manually.

**Reply by iuryt** [reply]
I also vote for that name!

**Comment by johnryantaylor** [comment]
I suppose that we could have it not capitalized, e.g. OceanBiome.jl if that is their convention.
